src/libs/kata-types/src/mount.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2019-2021 Alibaba Cloud
+// Copyright (c) 2019-2021 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Prefix to mark a volume as Kata special.
+pub const KATA_VOLUME_TYPE_PREFIX: &str = "kata:";
+
+/// The Mount should be ignored by the host and handled by the guest.
+pub const KATA_GUEST_MOUNT_PREFIX: &str = "kata:guest-mount:";
+
+/// KATA_EPHEMERAL_DEV_TYPE creates a tmpfs backed volume for sharing files between containers.
+pub const KATA_EPHEMERAL_VOLUME_TYPE: &str = "kata:ephemeral";
+
+/// KATA_HOST_DIR_TYPE use for host empty dir
+pub const KATA_HOST_DIR_VOLUME_TYPE: &str = "kata:hostdir";
+
+/// Information about a mount.
+#[derive(Debug, Clone, Default, serde::Serialize, serde::Deserialize)]
+pub struct Mount {
+    /// A device name, but can also be a file or directory name for bind mounts or a dummy.
+    /// Path values for bind mounts are either absolute or relative to the bundle. A mount is a
+    /// bind mount if it has either bind or rbind in the options.
+    pub source: String,
```

Comment:
How about using `PathBuf` for `source`, `destination` and `host_shared_fs_path`?

Reply:
`destination` and `host_shared_fs_path` are paths, but `source` may be a path a device name. So let's keep `source` as `String`?